distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -175,28 +183,86 @@ public void testCswRegistryCreateDup() throws Exception {
                 .post(CSW_PATH.getUrl());
     }
 
-    @Ignore
     @Test
     public void testCswRegistryStoreCreate() throws Exception {
-        //Stub test, waiting for additional features/service before implementing
-        //Use federation admin to create a registry entry
-        //on a 'remote' (aka loopback) catalog store
+        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
```

Comment:
That is fine. So you'd want to do the setup that you need for this test in an `@Before` if possible.